kolibri/core/assets/src/vue/session-nav-widget/login-modal.vue

```diff
@@ -2,7 +2,7 @@
 
   <div>
     <modal>
-      <div class="title" aria-labelledby="loginModal" slot="header">
+      <div class="title" aria-label="Log in to Kolibri" slot="header">
```

Comment:
@rtibbles This is the only change that will require string-wrapping, but at this point I'm not sure if I should wait for your PR to be merged, try wrapping only this myself or something else...?